packages/victory-scatter/src/index.d.ts

```diff
@@ -28,10 +29,14 @@ export interface VictoryScatterProps
   events?: EventPropTypeInterface<"data" | "labels" | "parent", StringOrNumberOrCallback>[];
   eventKey?: StringOrNumberOrCallback;
   maxBubbleSize?: number;
+  minBubbleSize?: number;
+  origin?: OriginType;
   samples?: number;
   size?: number | { (data: any): number };
+  sortOrder?: string;
   style?: VictoryStyleInterface;
   symbol?: ScatterSymbolType | { (data: any): ScatterSymbolType };
```

Comment:
can you check on `ScatterSymbolType` for me? I think it might be incomplete for recent props changes.

Reply:
On it!